google-mysql.yml

```diff
@@ -207,7 +207,12 @@ provision:
     details: Client private key for establishing a mutual TLS connection with the database instance.
 bind:
   plan_inputs: []
-  user_inputs: []
+  user_inputs:
```

Comment:
I am failing to understand the use case to allow insecure connections only for some bindings.

In my mind, we could come across production instances which don't allow insecure connections at all, or dev instances which can allow insecure connections.

Allowing insecure connections at binding time when an instance was provisioned to not allow for them, either explecitely or relying on default behaviour doesn't seem secure. I would make this a provision time parameter that then can be passed as computed input to the binding, or add a provision time input that can choose to enable or disable this capability at binding time.

Reply:
Thanks @pivotal-marcela-campo